services/mock-service/src/fsp-integration/safaricom/safaricom.mock.service.ts

```diff
@@ -51,14 +56,15 @@ export class SafaricomMockService {
           : 'Mock error message',
```

Comment:
Is this actually correct? In the mock scenarios errorOnCallback and errorOnCallbackForTimeout, the initial response is simply successful right. Basically in this piece of code success can always be assumed, because in errorOnRequest we already returned earlier.